recipe/meta.yaml

```diff
@@ -36,6 +36,10 @@ build:
   {% endif %}
   string: {{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}
 
+  run_exports:
+    # https://abi-laboratory.pro/index.php?view=timeline&l=netcdf-fortran
+    - {{ pin_subpackage('netcdf-fortran', max_pin='x.x') }} {{ build_pin }}
+
```

Comment:
@conda-forge/netcdf-fortran, does this seem right to all of you?

Reply:
I'm following:
https://github.com/conda-forge/libnetcdf-feedstock/blob/7682e750c11aab024167d75a7c4830e74964d163/recipe/meta.yaml#L49-L57

Reply:
I could not find info on the ABI laboratory. @isuruf what do you recommend?

Reply:
x.x is fine according to the ABI laboratory URL above

Reply:
Ouch! My dark theme hid that text as a faith grey and I could no read it. I guess that the oficial "bright" dark theme from GH is not that bad. I'll have to rever to it.